src/ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh

```diff
@@ -20,9 +20,10 @@ cd ../gcc-build
     --enable-checking=release \
     --disable-bootstrap \
     --disable-multilib \
-    --prefix=$(pwd)/../gcc-install
-make
-make install
+    --prefix=$(pwd)/../gcc-install > /dev/null
+
+make > /dev/null
+make install > /dev/null
```

Comment:
Are *all* of these steps generating thousands of lines?

Also please add a comment explaining why we're taking such drastic measures.

Reply:
The configure (if we added back `--quiet`) might give only a dozen lines, but `make` will definitely generate thousands of lines (even with `--quiet`). For `make install`, it might be only a couple hundred lines with `--quiet`.

Reply:
I still think it's way too much (for all commands). So better keep it this way as it's not relevant to the build.

I'll add comments.

Reply:
> The configure (if we added back --quiet) might give only a dozen lines,

Seems reasonable to just keep that visible then.

Reply:
Ok, done.

Reply:
You forgot to add back `--quiet` to configure, no?

Reply:
Not sure it changes anything but let's put it back.